### PR TITLE
styled the vehicle sheet

### PR DESCRIPTION
**Pull request by guswadsworth12**
Styled the Vehicle Sheet.

### CODE REVIEW

File: templates/actor/parts/actor-npc-defenses.hbs [hunk 1]
```diff
@@ -1,5 +1,5 @@
-<div style="flex-grow: 0;">
-  <div class="npc-defenses">
+<div class="stats-container " style="flex-grow: 0; height: 100%;">
```
Comment (phildominguez) [inline templates/actor/parts/actor-npc-defenses.hbs:1]: Extra space

**Review by phildominguez — CHANGES_REQUESTED** [review]
image
Noticing the movement container height doesn't match the others. Could this be fixed?

File: templates/actor/parts/actor-npc-defenses.hbs (hunk 1 above)
Comment (guswadsworth12) [reply templates/actor/parts/actor-npc-defenses.hbs]: fixed

File: templates/actor/parts/actor-common.hbs [hunk 2]
```diff
@@ -6,7 +6,7 @@
 
   {{!-- Movement --}}
   {{> "systems/essence20/templates/actor/parts/actor-movement.hbs"}}
-
+
```
Comment (phildominguez) [inline templates/actor/parts/actor-common.hbs:9]: Remove

**Review by phildominguez — CHANGES_REQUESTED** [review]

**Review by phildominguez — APPROVED** [review]
🔥

**Pull request merged**